### ISSUE TITLE
difference of values

### ISSUE THREAD

**Issue by gunjanvora** [dchester/jsonpath#100]
const cities = [
  { name: "London", "population": 8615246 },
  { name: "Berlin", "population": 3517424 },
  { name: "Madrid", "population": 3165235 },
  { name: "Rome",   "population": 2870528 }
];

Can i calculate the difference of populations of London and Berlin using jsonpath?

**Comment by RobertoFranco** [comment]
I have understood that jsonpath is only to retrive data, not to perform operations.

**Comment by sdawood** [comment]
I agree with Robert. Check 'json-tots' npm package for JSON document level operations and JSON transformation
